projects/plugins/crm/src/js/components/automations/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { __ } from '@wordpress/i18n';
+
+import './styles.scss';
+
+export const AutomationsAdmin = () => {
+	return (
+		<div className="jpcrm-automations-admin__container">
```

Comment:
Should we perhaps reuse the `<AdminPage>` RNA component since it makes sure we show a footer etc? 😄

If we use the `AdminPage` component that already exist in CRM (which wraps the RNA component) we should be able to only define e.g. the footer in 1 location.

**References**:

* `projects/plugins/crm/src/js/components/admin-page/index.tsx`
* `projects/js-packages/components/components/admin-page/index.tsx`

Reply:
In this case we already have the header and footer:

<img width="1168" alt="image" src="https://github.com/Automattic/jetpack/assets/42627630/e7719ecf-4a9f-41a1-b37e-d73f808a8228">

Those are both existing functionality written in PHP and added to the page by CRM, so if we switch to the RNA component then we would have to recreate it in React and also change CRM to not render this header and footer for the automations page. This could be something worth doing, especially if we want to start making more pages in React, but I think it's a project on its own.

Reply:
For reference, this is what it looks like when I use `AdminPage` without making any other changes to CRM. The headers and footers are doubled.

<img width="1167" alt="image" src="https://github.com/Automattic/jetpack/assets/42627630/f7794c6d-9376-4398-8e2e-56a100ad2184">

Reply:
The admin component accepts `showHeader` and `showFooter` props to hide them. I've gone ahead and made a small refactor so it's reusable for future admin pages as well.